packages/types/src/fulfillment/service.ts

```diff
@@ -337,6 +351,18 @@ export interface IFulfillmentModuleService extends IModuleService {
     sharedContext?: Context
   ): Promise<ShippingOptionTypeDTO>
 
+  /**
```

Comment:
NIT: Grouping the methods by model, rather than by function (eg. you put retrieve, list, create, and update next to each other, followed by retrieveGeoZone, listGeoZone...) makes it a bit easier to find things